draft-irtf-cfrg-vdaf.md

```diff
@@ -6359,8 +6374,38 @@ defined below.
 `agg_result`:
 : The expected aggregate result of type `AggResult` defined by the VDAF.
 
-The schema also includes whatever parameters are required to instantiate the
-VDAF. These are listed in the subsections below.
+`operations`:
+:  A list of operations used to construct the test case. They are executed
```

Comment:
@divergentdave I guess we also need to define what it means to "execute" an operation. Any suggestions?